openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_render_settings.json

```
@@ -145,7 +145,7 @@
                 },
                 {
                   "type": "label",
-                  "label": "Add additional options - put attribute and value, like <code>defaultArnoldRenderOptions.AASamples</code> = <code>4</code>"
+                  "label": "Add additional options - put attribute and value, like <code>defaultArnoldRenderOptions.AASamples</code> for attribute and to next field value <code>4</code>"
```

Doesn't feel that much clearer to me.

```suggestion
                  "label": "Add additional options - set the attribute and value, like <code>defaultArnoldRenderOptions.AASamples</code> with the node name with attribute left and the required value to the right <code>4</code>"
```
Not sure if the UI is left right like that, but that reads clearer to me at least.

Also, are these labels also in AYON settings? If so, might need updating there too?